torch/nn/_functions/rnn.py

```diff
@@ -21,8 +21,21 @@ def RNNTanhCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None):
     return hy
 
 
+def _match_hidden_with_input(input, hidden):
+    '''The CUDNN backend doesn't support broadcasting, so this function
+    broadcasts the hidden batch size to match the input batch size, if
+    possible'''
+    if isinstance(hidden, tuple):
```

Comment:
This comment is misleading, it's not cudnn backend, it is fused LSTM and GRU kernels in pytorch/THCUNN.

Reply:
Thanks for pointing that out, will update!